apps/ledger-live-desktop/tests/models/CustomImageDrawer.ts

```diff
@@ -0,0 +1,133 @@
+import { Page, Locator } from "@playwright/test";
+
+async function waitFor(
+  predicate: () => Promise<boolean>,
+  intervalMs = 100,
+  timeout = 10000,
+): Promise<boolean> {
```

Comment:
it's a nice function, maybe put in a file inside `apps/ledger-live-desktop/tests/utils/` ?